Engine/Component/ComponentTrail.h

```diff
@@ -81,7 +81,7 @@ class ComponentTrail : public Component
 
 	//Color properties
 	float color[4] = { 1.0f, 1.0f, 1.0f, 1.0f };
```

Comment:
Store the color as a `float4` instead as a `float[4]` and everything will be simplified in `SpecializedLoad` and `SpecializedSave`, and also to keep everything consistent